README.md

```diff
@@ -1 +1,87 @@
-# rogueserver
\ No newline at end of file
+# rogueserver
+
+# Hosting in Docker
+It is advised that you host this in a docker container as it will be much easier to manage. 
+There is a sample docker-compose file for setting up a docker container to setup this server.
+
+# Self Hosting outside of Docker:
+Recommended Tools:
+If using Windows: [Chocolatey](https://chocolatey.org/install)
+
+## Required Tools:
+- Golang
+- Node: **18.3.0**
+- npm: [how to install](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+
+## Installation:
+The docker compose file should automatically implement a container with mariadb with an empty database and the default user and password combo of pokerogue:pokerogue
+
+### src/utils.ts:224-225 (in pokerogue)
+Replace both URLs (one on each line) with the local API server address from rogueserver.go (0.0.0.0:8001) (or whatever port you picked)
+
+# If you are on Windows
+
+Now that all of the files are configured: start up powershell as administrator:
+```
+powershell -ep bypass
```

Comment:
Is this actually needed here?

Reply:
At the time of writing, I had seen others bringing up issues with npm being run due to executionpolicy being set to AllSigned or Restricted. Since the program is intended to be run by the User, Execution policy would just get in the way. This is a step to reduce people from asking about why there is yet another error message. Execution policy prevents malicious code from accidentally being ran. If this needs to be removed, then we can.

Reply:
Yes but this is the block for the `go` executable, not the npm powershell script.